### ISSUE TITLE
PropTypes issue

### ISSUE THREAD

**Issue by egorovsa** [yisheng90/react-loading#9]
You have propTypes issues, did not you get any warnings in browser console?

your version:

```
  height: PropTypes.oneOfType[(PropTypes.string, PropTypes.number)],
  width: PropTypes.oneOfType[(PropTypes.string, PropTypes.number)]
```

must be:

```
  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
```

pay attention on brackets 

you have `[(`, must be ([

**Comment by egorovsa** [comment]
And yes.. Remove the old Skeleton.d.ts 
you don't need this anymore.
Thanks!

**Comment by egorovsa** [comment]
Check also please if you have params those are required.
It seems to me I had case when I had't passed params and got error.
Therefor if any params really required set them .isRequired. in propTypes
and remove  ? from d.ts for those params.

**Comment by yisheng90** [comment]
Resolved at version 1.4.4
